group/add-member/schema.json

```diff
@@ -60,15 +60,15 @@
   "definitions": {
     "messageId": {
       "type": "string",
-      "pattern": "^%[a-zA-Z0-9\\/+]{42}[AEIMQUYcgkosw048]=.sha256$"
+      "pattern": "^ssb:message\/classic\/[a-zA-Z0-9_\\-]{43}[AEIMQUYcgkosw048]=$"
     },
-    "cloakedMessageId": {
+    "groupId": {
       "type": "string",
-      "pattern": "^%[a-zA-Z0-9\\/+]{42}[AEIMQUYcgkosw048]=.cloaked$"
+      "pattern": "^ssb:identity/group/[a-zA-Z0-9_\\-]{43}[AEIMQUYcgkosw048]=$"
```

Comment:
I escaped the `-` with 1 backslash at first because we do that here

https://github.com/ssbc/ssb-uri2/blob/00151c68b6972dbfb60393aa710244a8a06b191a/src/index.ts#L270

but then vscode complained about

```
Invalid escape character in string. json(261)
```

Adding another backslash like we have on line 22 in this file made it stop complaining. Is it actually correct like this?

Reply:
:shrug:

Reply:
Test the pattern on a expected failing + passing case.
testing!

where is there a 43 instead of 42?

Reply:
kept the 2 backslashes and added a test that backslashes aren't allowed